lib/jekyll/tags/strapiimagefilter.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require "down"
+
+module Jekyll
+      class StrapiStaticFile < StaticFile
+        def initialize(site, base, dir, name, collection = nil)
+          @site = site
+          @base = base
+          @dir  = dir
+          @name = name
+          @collection = collection
+          # TODO: Check if user can change 'assets' path
+          @relative_path = File.join(*["assets", @name].compact)
+          @extname = File.extname(@name)
+        end
+      end
+
+    module StrapiImageFilter
+      def asset_url(input)
+        assets_path = "assets" # TODO: to figure our in which conditions it can change
```

Comment:
**assets_path** is this path specific to the _site folder or the root of the jekyll workspace? 

The _site folder for all my jekyll projects are: **assets** >> ***images***

This is a bit out of context, but I also have an **uploads** folder that is specific to **collection** images only like posts andor (xor) products.

Reply:
Your default path "assets/images" should be transparent for filter - filter generates new url as return  of the function, so it should adapt... Perhaps I can add more debug in the filter!